Lib/argparse.py

```diff
@@ -2470,12 +2470,18 @@ def parse_known_intermixed_args(self, args=None, namespace=None):
     # Value conversion methods
     # ========================
     def _get_values(self, action, arg_strings):
-        # for everything but PARSER, REMAINDER args, strip out first '--'
         if not action.option_strings and action.nargs not in [PARSER, REMAINDER]:
```

Comment:
Maybe you could break down each binary expression into smaller chunks. It seems a bit too long otherwise.
```
if not action.option_strings
   and action.nargs
   not in [PARSER, REMAINDER]:
```

Reply:
I haven't modified this line in the PR, so I'll keep it as is unless the PR receives feedback requiring more substantial refactoring. 

As an aside, the above suggestion would also require parens to be valid syntax.

Reply:
> I haven't modified this line in the PR, so I'll keep it as is unless the PR receives feedback requiring more substantial refactoring.
> 
> As an aside, the above suggestion would also require parens to be valid syntax.

Oh yes, I almost forgot.